tools/sphere.c

```diff
@@ -0,0 +1,108 @@
+// Mystical Sphere of Slooping
+// Gon
+// 2017-04
+
+inherit NewThingCode;
+inherit AnsiTellCode;
+inherit "/usr/gon/tools/sloop";
```

Comment:
I've got this created, but not sure if sloop.c needs it as well? I'm not doing any inherits in sloop.c, so for now I've only put it in sphere.c as I wasn't sure what the ramifications for sloop.c inheriting NewThingCode or AnsiTellCode would be? I'll upload common.h when I commit all my changes.